src/views/TradingCompetition/components/Sticker.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components'
+import { Flex } from '@pancakeswap-libs/uikit'
+
+interface StickerProps {
+  backgroundColor?: string
+  borderColor?: string
+}
+
+const Sticker = styled(Flex)<StickerProps>`
```

Comment:
This is just a circular flex with a border & bg colour to wrap around circular images that need a 'sticker' border and drop shadow around them, i.e.

<img width="334" alt="Screenshot 2021-03-19 at 16 32 00" src="https://user-images.githubusercontent.com/79279477/111813123-b53a3d00-88d0-11eb-8e63-4ac7f3348693.png">

<img width="281" alt="Screenshot 2021-03-19 at 16 32 13" src="https://user-images.githubusercontent.com/79279477/111813121-b4a1a680-88d0-11eb-9c16-98a52c111241.png">